WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java

```diff
@@ -265,6 +265,7 @@ public static void populateTextFieldWithin(Integer elementID, String text) {
 
     public static void populateTextField(ViewInteraction element, String text) {
         waitForElementToBeDisplayed(element);
+        clickOn(element);
```

Comment:
Looks like there is also `waitForElementToBeDisplayed(viewInteraction);` in `clickOn()`. Should we remove the one from line 267 since it's going to be called twice with this change?

Reply:
Good point @jostnes! While having it this way should almost not hurt if the element exists, it will consume twice as much time if it doesn't. Applied your suggestion in 77c3c0e748fa0f4d2c2976dcf7f49377229aac7f.